### ISSUE TITLE
PUT isfavorite not taken into account

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2305]
Branch 18.06

PUT https://fqdn.fr/console/private/roles/EL_TOTO 
```json
{"description":null,"cn":"EL_TOTO","users":[],"isFavorite":false,"originalID":"EL_TOTO"}
```

GET https://fqdn.fr/console/private/roles/EL_TOTO 
```json
{"description":null,"cn":"EL_TOTO","users":[],"isFavorite":true}
```

**Comment by groldan** [comment]
Fixed on 18.06 by means of #2404
Awaiting merge of #2403 for a fix on master including integration testing

**Comment by groldan** [comment]
Closing, fixed by #2403
